flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/dialect/AbstractDialect.java

```diff
@@ -41,228 +28,11 @@
  *
  * <p>Implementors should be careful to check the default SQL statements are performant for their
  * specific dialect and override them if necessary.
+ *
+ * @deprecated use org.apache.flink.connector.jdbc.core.table.dialect.AbstractDialect
```

Comment:
```suggestion
 * @deprecated Use {@link org.apache.flink.connector.jdbc.core.database.dialect.AbstractDialect}.
```
How about using the 'link' tag to point the target new Class ?

Reply:
done